content/docs/learning-guide/tutorials/remote-repositories/create-remote-repository.md

```diff
@@ -0,0 +1,177 @@
+---
+linktitle: Remote Repositories
+title: Create Your Remote Repository
+description: Learn how to use remote repository templates in Open 3D Engine (O3DE) to package and share your project remotely with GitHub or other version control platforms.
+---
+In this tutorial you will build upon the foundation of O3DE and use the remote repository template to manage and distribute projects effortlessly. You will learn how to package and share your projects remotely with GitHub or other version control platforms, promoting collaboration and accelerating project development.
+
+Show a list Open 3D Engine (O3DE) commands.
+```
+o3de.bat --help
+```
+**NOTE:** All engine commands have their abbreviations. Use the `--help` menu for list of details.
+
+## Create a repository
+Create a new repository for your **Remote project** and clone this repo to your local machine.
+This version control repository link is where you upload your project `repo-uri`.
+
+Here is an example of a Github repo uri:
+
+```
+"https://github.com/<YourGitAccount>/<RemoteProjectName>.git"
+```
+
+## Create Remote Repository Configuration
+Navigate to scripts folder from the root of your **Open 3D Engine (O3DE)**.
+```
+cd scripts
+```
+To create a remote repository configuration, you will need to provide the following information:
+- `repo-path` - the local path of your remote project. This is where you cloned your repo.
+- `repo-uri` - the URI of your repo.
+
+{{< tabs name="O3DE CLI command" >}}
+{{% tab name="Windows" %}}
+```cmd
+o3de.bat create-repo --repo-path "C:\o3de_Projects\<RemoteProjectName>" --repo-uri "https://github.com/<YourGitAccount>/<RemoteProjectName>.git"
+```
+**NOTE:** Use ./o3de.bat if you are using powershell
+
+{{% /tab %}}
+{{% tab name="Linux" %}}
+```
+./o3de.sh create-repo --repo-path "~/<RemoteProjectName>" --repo-uri "https://github.com/<YourGitAccount>/<RemoteProjectName>.git"
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+The `create-repo` command performs the following actions:
+1. Creates a directory with the remote project name specified in the `repo-path`, and creates a `repo.json` file in the root of that directory.
+2. Creates three folders in your project folder: Gems, Projects, and Templates. Make sure to place all associated objects in their correct folder type to avoid later confusion.
```

Comment:
```suggestion
2. Creates three folders in your remote repository folder: Gems, Projects, and Templates. Make sure to place all associated objects in their correct folder type to avoid confusion later.
```